### DISCUSSION TITLE
JsonBodyMiddleware and RawBodyMiddleware runtime warnings

### DISCUSSION THREAD

**Discussion by kachar** [golevelup/nestjs#385] [Q&A] [answered]
During the start times of the application, the following warnings have been raised:

> WARN [DependenciesScanner] In the next major version, Nest will not allow classes annotated with @Injectable(), @catch(), and @controller() decorators to appear in the "imports" array of a module.
Please remove "JsonBodyMiddleware" (including forwarded occurrences, if any) from all of the "imports" arrays.

> WARN [DependenciesScanner] In the next major version, Nest will not allow classes annotated with @Injectable(), @catch(), and @controller() decorators to appear in the "imports" array of a module.
Please remove "RawBodyMiddleware" (including forwarded occurrences, if any) from all of the "imports" arrays.

> Scope [AppModule]

The warnings are triggered when we integrate `webhooks` package according to the documentation:

https://github.com/golevelup/nestjs/blob/ad7cd398280ade41aef58c3c7a642641b434aa24/packages/webhooks/README.md#step-2-include-middlewares-in-your-appmodule

Is there's a better way to integrate the module so we're tuned to the latest NextJS expectations?

Related to https://github.com/podkrepi-bg/api/issues/70

**Comment by kachar** [comment]
Any update on this one as it's been a month since the discussion is opened?

**Comment by munky69rock** [comment, marked as answer]
Perhaps these middleware imports are no longer needed.
https://github.com/golevelup/nestjs/commit/cadaf1e3bc62dfc4c8e1809f1e1057f274fa9c4a
